src/main/java/edu/umich/ctools/sectionsUtilityTool/Utils.java

```diff
@@ -58,6 +77,8 @@
 	public static final String CANVAS_API_VERSION = "/api/v1" ;
 	public static String canvasURL = null;
 	public static String canvasToken = null;
+	public final static int API_UNKNOWN_ERROR = 666;
```

Comment:
Does this mean "error because the API is unknown" or "The API had an unknown error"?